File: user-api/go-sdk/pkg/models/operations/inviteuser.go
package operations

import (
	"openapi/pkg/models/shared"
)

type InviteUserRequest struct {
	Request *shared.UserInvitationPayload `request:"mediaType=application/json"`
}

type InviteUserResponse struct {
	ContentType string
	StatusCode  int64
	UserV2      *shared.UserV2
}

File: user-api/ts-sdk/src/sdk/models/operations/inviteuser.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class InviteUserRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "request, media_type=application/json" })
  request?: shared.UserInvitationPayload;
}


export class InviteUserResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  userV2?: shared.UserV2;
}
